### ISSUE TITLE
Listing profiles for community roles

### ISSUE THREAD

**Issue by maneesha** [carpentries/new-website#79]
Toby recommended the following as a FAQ style question on our website, instead of repeating this language across our various profile pages.  I think this is an excellent idea.  Opening this issue to think about where this should go, since we do not have a general FAQ.

> Question: Why doesn’t my profile appear on the Instructors, Maintainers, and/or Trainers pages? 
> Answer: These pages list community members who have consented to appear on our website. If you are a badged Instructor, Maintainer, and/or Trainer who is not listed here but would like to be, please [update your profile in AMY (our internal database)](https://amy.carpentries.org/) and agree to make your profile public. Log in with GitHub and save your changes. If you have difficulty logging in, please email us. [Read more about how these profiles are published](https://carpentries.org/community-member-profile-faq/).

**Comment by maneesha** [comment]
Related question - what else could be in a general site-wide FAQ?

**Comment by froggleston** [comment]
The text reads well. This could be in an accordion or at the bottom of the relevant page? If not a site FAQ would be good.

As for what else could be in a FAQ, we could at least link to particular other FAQs we mention, e.g. the instructor FAQ. However, having a "Carpentries-wide" FAQ might be a really nice addition to the new site.

**Comment by maneesha** [comment]
Assigning to @OscarSiba as we chatted yesterday about what a Carpentries-wide FAQ would look like. This would be distinct from specific FAQs we already have (for workshops, membership, etc.)

**Comment by elletjies** [comment]
We have a couple of FAQ pages I can think of (membership, instructor etc). Is there a way to have these linked into a site wide FAQ page

**Comment by sheraaronhurt** [comment]
Yes. Maybe the FAQ could be categorized by Member, Instructor, Trainer, General??

**Comment by maneesha** [comment]
Renamed issue because this is about a potential site-wide FAQ rather than just about community profiles

**Comment by OscarSiba** [comment]
This we can only really complete once the Handbook is also in place, as some of the answers in a Carpentries-wide FAQ would also link to it, and possibly also to materials to be placed in the Commons. Might be a page worth revisiting in the January iteration of the website. Alternatively we can work towards one to launch at the same time as the Handbook.

**Comment by maneesha** [comment]
Discussed with @OscarSiba - move to Q3

**Comment by maneesha** [comment]
Discussed with @OscarSiba - move to Q4
